### DISCUSSION TITLE
Welcome to my Discussions!

### DISCUSSION THREAD

**Discussion by mrmaxguns** [mrmaxguns/mrmaxguns#1] [General]
## 👋 Welcome!

I'm trying out GitHub discussions as a way to communicate. If you have a question or comment for me, you've come to the right place. If you have a minor comment, or question, just reply down below. Otherwise, open a new discussion so that we can communicate. Note that these discussions are public, and with regard to programming. If you want to talk to me about something else, please email me.

**Comment by jack-davidson** [comment]
Hi! I am Jack Davidson. I am in advanced coding B5 and stumbled upon your profile. I saw you are learning C and I am learning it as well and if you wanted to discuss you can join my discord server [https://discord.gg/2vCMACHX]() Thanks.

**Reply by mrmaxguns** [reply]
I joined. Thanks for reaching out!

**Comment by jack-davidson** [comment]
(:
